.github/workflows/docker-publish-ppml.yaml

```diff
@@ -1,6 +1,10 @@
 name: Nightly Build Docker Publish BigDL-PPML
 
 on:
+  pull_request:
```

Comment:
this change may affect the other ppml images
you can add this to your pr to gramine to trigger the image building but never merge it to the main branch